FastSimulation/Tracking/python/GlobalPixelSeedProducerForElectrons_cff.py

```diff
@@ -44,9 +43,7 @@
     'FPix1_pos+FPix2_pos',
     'FPix1_neg+FPix2_neg',
 )
-globalPixelSeedsForPhotons.skipSimTrackIdTags = [cms.InputTag("globalPixelStepIds")]
-globalPixelSeedsForPhotons.outputSeedCollectionName = 'GlobalPixel'
-globalPixelSeedsForPhotons.zVertexConstraint = cms.double(-1)
+globalPixelSeedsForPhotons.simTrackSelection.skipSimTrackIds = [cms.InputTag("globalPixelStepIds")]
 globalPixelSeedsForPhotons.originRadius = cms.double(0.02)
 globalPixelSeedsForPhotons.originpTMin = cms.double(1.5)
```

Comment:
* values for originRadius, originpTMin -> no clue... it would be best to import it from somewhere as well
* zVertexConstraint was buggy and never properly used